### PR TITLE
Fix outbox processing

### PR DESCRIPTION
**Pull request by manueliglesias**

### CODE REVIEW

**Comment by undefobj** [comment]
Looks good.

File: packages/aws-appsync/src/link/offline-link.js [hunk 1]
```diff
@@ -46,17 +46,23 @@ export class OfflineLink extends ApolloLink {
...
-                    observer.complete();
+                if (!doIt) {
+                    if (!optimisticResponse) {
+                        console.warn('An optimisticResponse was not provided, it is required when using offline capabilities.');
 
-                    return () => null;
-                } else {
-                    // console.log('Processing mutation');
+                        if (!online) {
+                            throw new Error('Missing optimisticReponse while offline.');
```
Comment (elorzafe) [inline packages/aws-appsync/src/link/offline-link.js]: typo optimisticResponse

Maybe I would add a code comment something like 
// offline cache without optimistic response is processed immediately

**Comment by manueliglesias** [comment]
Fixes #30

**Comment by rivertam** [comment]
Thank you! Not familiar with how this library versions and stuff. When will this be available on npm?

**Comment by elorzafe** [comment]
Hi rivertam,

The update was published yesterday. 

The package name on npm is aws-appsync

**Pull request merged**